# Assignment-2-Pt-2
#Camilo Ortiz
#Data Structures 303
October 18, 2022

This assignment consists of two parts where the first part relates to the exception handling of two functions in the program. That gives more leway to the
user and program if it were to recieve and out of bounds input that did not work. Resulting in the throw and catch of the error or input. 

The class program evaluates and determines the salary of professional and non-professional employees. 
I established a base class then two other derived classes from that class to split it in two new sections. The program runs through both instances when ran
that asks for the data needed for find the salary, vacation days and insurance plan that corresponds to the employee. 
User inputs the data needed and the program runs data through functions in derived classes.

<img width="622" alt="Screen Shot 2022-10-18 at 1 17 43 PM" src="https://user-images.githubusercontent.com/78284890/196537275-6f75cb5d-a4cd-4647-b222-26156cb0043a.png">
<img width="611" alt="Screen Shot 2022-10-18 at 1 18 02 PM" src="https://user-images.githubusercontent.com/78284890/196537282-1c8d4634-4b8f-4df9-aca3-0a7a0a1da1d1.png">
